src/TestFramework/PhpUnit/Adapter/PestAdapterFactory.php

```diff
@@ -70,7 +72,8 @@ public static function create(
         array $sourceDirectories,
         bool $skipCoverage,
         bool $executeOnlyCoveringTestCases = false,
-        array $filteredSourceFilesToMutate = []
```

Comment:
unrelated, but we could update the CS fixer to append the trailing commas all the time to avoid this sort of diff

Reply:
agree, extracted to https://github.com/infection/infection/issues/1920